pad_api_data/pad_etl/processor/skill_info.py

```diff
@@ -559,7 +566,7 @@ def f(x):
                 color_text = 'all' if len(for_attr) == 5 else ', '.join(
                     [ATTRIBUTES[i] for i in for_attr])
                 for_skill_text += ' ' + color_text
-                c['skill_text'] += 'Enhance all' + for_skill_text + ' orbs'
+                c['skill_text'] += 'Enhance ' + for_skill_text + ' orbs'

```

Comment:
Can you back out this change for now and open a separate PR?

If I do this it's going to update a ton of monsters and I would want to rebuild the first-load database afterwards.